tests/beakerlib/Traceback-caused-by-scheduler-plugin-with/runtest.sh

```diff
@@ -29,33 +28,36 @@ rlJournalStart
         rlServiceStart "tuned"
         rlImport "tuned/basic"
         tunedProfileBackup
-	rlFileBackup "/etc/tuned/active_profile"
+        rlFileBackup "/etc/tuned/active_profile"
         rlFileBackup "/etc/tuned/profile_mode"
 
-        rlRun "mkdir $PROFILE_DIR/test-profile"
-        rlServiceStart "tuned"
+        PROFILE_DIR=$(tunedGetProfilesBaseDir)
+
+        rlRun "mkdir -p ${PROFILE_DIR}/test-profile"
+        rlServiceStop "tuned"
         sleep 1
-	rlFileBackup "/var/log/tuned/tuned.log"
-	rlRun "rm -rf /var/log/tuned/tuned.log"
+        rlFileBackup "/var/log/tuned/tuned.log"
+        rlRun "rm -rf /var/log/tuned/tuned.log"
+        rlServiceStart "tuned"
```

Comment:
Why separate the service stop? I'd back up the logs first and then run rlServiceStart (which restarts TuneD if it's already running).

Reply:
It's taken from the RHEL dist-git upstream Beaker test. I wanted to sync to their version. IMHO it's trying to avoid possible race if the daemon is simultaneously logging while the file is removed/backed-up. Without it IMHO the worst case that could happen is that for multiline logs there would be some lines missing in the log (IMHO it uses line buffering). But the calls in the test could be reordered.

Reply:
Ok, let's keep the tests in sync.